### PR TITLE
[IOTDB-4156] import csv has error time after timestamp_precision set ns

### PR DESCRIPTION
**Pull request by cmlmakahts**
Add a option "tp" to specify timestamp precision (ms/ns/us)

image

### CODE REVIEW

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 1]
```diff
@@ -82,6 +82,9 @@ public class ImportCsv extends AbstractCsvTool {
   private static final String CSV_SUFFIXS = "csv";
   private static final String TXT_SUFFIXS = "txt";
 
+  private static final String TIMESTAMP_PRECISION_ARGS = "tp";
+  private static final String TIMESTAMP_PRECISION_NAME = "timestamp precision (ms/us/ns)";
+
```
Comment (HTHou) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:87]: Consider updating the user guide?

File: cli/pom.xml [hunk 2]
```diff
@@ -45,7 +45,6 @@
             <artifactId>iotdb-server</artifactId>
             <version>${project.version}</version>
             <type>test-jar</type>
-            <scope>test</scope>
```
Comment (wangchao316) [inline cli/pom.xml:48]: The client cannot directly import iotdb-server and jar. need deal...

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java (hunk 1 above)
Comment (cmlmakahts) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:87]: User guide is updated.

File: cli/pom.xml (hunk 2 above)
Comment (cmlmakahts) [reply cli/pom.xml]: Using shade plugin shaded DatetimeUtils.class into iotdb-client.jar

**Comment by HTHou** [comment]
Some tests failed, PTAL

File: docs/UserGuide/Write-And-Delete-Data/CSV-Tool.md [hunk 3]
```diff
@@ -174,9 +174,9 @@ Time,Device,str(TEXT),int(INT32)
 
 ```shell
 # Unix/OS X
-> tools/import-csv.sh -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv> [-fd <./failedDirectory>] [-aligned <true>]
+> tools/import-csv.sh -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv> [-fd <./failedDirectory>] [-aligned <true>] [-tp <ms/ns/us>]
 # Windows
-> tools\import-csv.bat -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv> [-fd <./failedDirectory>] [-aligned <true>]
+> tools\import-csv.bat -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv> [-fd <./failedDirectory>] [-aligned <true>] [-tp <ms/ns/us>]
```
Comment (HTHou) [inline docs/UserGuide/Write-And-Delete-Data/CSV-Tool.md:179]: Update Chinese user guide as well?

Comment (cmlmakahts) [reply docs/UserGuide/Write-And-Delete-Data/CSV-Tool.md:179]: Fixed

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

**Pull request merged**